app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/actionableaddress/ActionableAddressView.kt

```diff
@@ -280,6 +280,7 @@ sealed interface ActionableAddress {
             is com.radixdlt.sargon.Address.Resource -> "resource"
             is com.radixdlt.sargon.Address.Validator -> "component"
             is com.radixdlt.sargon.Address.Vault -> null
+            is com.radixdlt.sargon.Address.Locker -> null
```

Comment:
I think in demo, Matt mentioned that we might need to also preview lockers as components in the "Using..." section. If that is the case, then this section, if I remember correctly shows the actionable addresses. In this case the addresses will be lockers, and will provide a link to dashboard. I think, if you have one available, to search the page the dashboard lands when you put on a locker address and set it up here too.